home.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/home.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <title>Mauritania palms</title>
+</head>
+<body>
+    <header class="header">
+        <img src="images/navBar.svg" alt="navBar" class="navBar">
+        <img src="images/closeIcon.svg" alt="closeIcon" class="closeIcon dis-none">
+        <div class="navUl dis-none" id="navUl">
+            <a href="./home.html" class="home upperCase">home</a>
+            <a href="./about.html" class="about upperCase">about</a>
+        </div>
+        <div class="desktopHeader">
+        <img src="./images/logo.jpg" alt="logo" class="logo">
+        <div class="desktopUl">
+            <a href="" class="desktopUl-li upperCase">about</a>
+            <a href="" class="desktopUl-li upperCase">sponsors</a>
+            <a href="" class="desktopUl-li upperCase">Persons</a>
+            <a href="" class="desktopUl-li upperCase">contact us</a>
+        </div>
```

Comment:
The top _dark navbar_ and the bottom _white navbar_ of your desktop version do not match the provided [template design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Kindly note we actually want our design to as much as possible _mimic_ the design of the provided template design. Kindly ensure to improve upon these aspects:
- [ ] The menu **fonts** are in lowercase, as can be seen in the original design below.
- [ ] There's a menu item with a thick red border on the extreme right of the navbar c


**Your header navigation sections:**
![image](https://user-images.githubusercontent.com/12745474/133655953-d9854da6-59dd-4eb9-9aeb-7baa455b8dcc.png)

<br>

**The original header navigation sections:**
![image](https://user-images.githubusercontent.com/12745474/132956330-393c697b-60c8-4a14-a9a8-0f7397ac04c9.png)

Reply:
Addressed this but you can still improve to look like the design